### ISSUE TITLE
Identify properties of already open swing app

### ISSUE THREAD

**Issue by SanjeevKumarmn** [nokia/jspy#12]
how to get object properties of swing app fields? Similar to UISPY.exe or inspect.exe or F12 developer options for browser. As a functional tester, i do not have access to the AUT code to pickup the swing app java class and instead to use a bat file or webstart to start a java think client. let me know jspy can identify the components of already opened swing app and I can refer this components properties for my functional automation testing tools.

**Comment by OanaG** [comment]
Yes, JSpy is used to identify the properties of the components of a java swing application. The application is started from JSpy.
JSpy needs the jar file of the application that you want to inspect. The source code is not necessary. The jar file of your application must be included in path.

**Comment by SanjeevKumarmn** [comment]
Thanks for the response. I was trying to use the below app to find object properties. 
https://github.com/jalian-systems/swingset3/blob/master/SwingSet3.jar
The app doesn't start through the jspy, it says connected and few seconds later disconnected.  I tried executing the jar file, as well as used command mode to run  and it dint work. It did not respond for CTR+ALT+R for any swing app that is already opened.

**Comment by OanaG** [comment]
If you can not execute the jar file it means that the problem is not from JSpy, but from your jar file. I have tried to launch SwingSet3.jar from command line, but it seams that it throws a NoClassDefFound exception. First you need to solve the problem with the jar file and then try again with JSpy.
